lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/update-address.yaml

```diff
@@ -0,0 +1,119 @@
+# Entry points
+
+START:
+  events:
+    next:
+      targetState: ADDRESS_AND_FRAUD_UPDATE_ADDRESS
+
+# Parent States
+
+CRI_STATE:
+  events:
+    not-found:
+      targetJourney: FAILED
+      targetState: FAILED_RFC
+    fail-with-no-ci:
+      targetJourney: FAILED
+      targetState: FAILED_RFC
+    error:
+      targetJourney: TECHNICAL_ERROR
+      targetState: ERROR
+    access-denied:
+      targetJourney: FAILED
+      targetState: FAILED_RFC
+    enhanced-verification:
+      targetJourney: FAILED
+      targetState: FAILED_RFC
+    temporarily-unavailable:
+      targetJourney: TECHNICAL_ERROR
+      targetState: ERROR
+    fail-with-ci:
+      targetJourney: FAILED
+      targetState: FAILED_RFC
+    vcs-not-correlated:
+      targetJourney: FAILED
+      targetState: FAILED_RFC
+    alternate-doc-invalid-dl:
+      targetJourney: FAILED
+      targetState: FAILED_RFC
```

Comment:
We don't seem to be using this parent state anywhere, I think we can probably remove it but it might be worth checking it still works without it

Reply:
@RogVanGDS I initially did not put it in but it was used in the nested ADDRESS_AND_FRAUD journey so did not work, see https://github.com/govuk-one-login/ipv-core-back/blob/77375cb41affb0869a124897acd47e30b8e316a2/lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/nested-journey-definitions.yaml#L1

Reply:
If we do need the parent state, should the failures go to `FAILED_ADDRESS` rather than `FAILED_RFC`? (If those are actually different)